app/javascript/admin/AdminUserActionsModal.jsx

```diff
@@ -0,0 +1,20 @@
+import { h } from 'preact';
```

Comment:
This isn't finished as I was just doing a proof of concept, but this should be solid. It still needs some docs and proptypes.